README.md

```diff
@@ -33,60 +42,61 @@ Add an html page that references the image. Below is the html we will use for th
-  * ajax.googleapis.com
-  * fonts.googleapis.com
-  * fonts.gstatic.com
-  * tpc.googlesyndication.com
-  * If you use of the ```<video>``` or ```<audio>``` tag you must host the external assets.
-* Videos that play by default MUST NOT play audio by default.
-* Audio MUST NOT play by default.
+
+In order for your HTML5 ad to pass validation, it must meet the following requirements:
+
+* Zip file must be 250 kb or smaller.
+* Zip file may contain up to 50 files.
+* There must be a single file named `index.html`.
+* HTML5 ad `index.html` must include a:
+  * ```<!DOCTYPE html>``` declaration.
+  * ```<html>``` tag.
+  * ```<head>``` tag.
+  * ```<body>``` tag.
+* Zip file may contain any of the other following file types:
+  * `.css`, `.js`, `.html`, `.gif`, `.png`, `.jpg`, `.jpeg`, `.svg`
+* Subfolders are not allowed.
+* HTML5 ad total render size must not exceed 2.1 MB.
+* External reference are allowed, but must not exceed 10.
+* External references cost against your overall render size limit; however, the following domains are exempt:
+  * `s0.2mdn.net/ads/`
+  * `ajax.googleapis.com`
+  * `fonts.googleapis.com`
+  * `fonts.gstatic.com`
+  * `tpc.googlesyndication.com`
+* If you use a `<video>` or `<audio>` tag you must host the external asset.
+* Videos that play by default **must not** play audio by default.
+* Audio **must not** play by default.
 
 ## API Usage
+
 HTML5 Assets are now available through the Ads endpoint and are returned as a collection of assets under the ad.
 
 HTML5 Assets are not directly accessible. If you make a mistake and need to change an asset after it has been uploaded just delete the ad and recreate it.
 
 
 ## Additional Information
 
-* If you include an ad.size meta tag in the ad when you upload it the system will automatically detect and choose the correct ad size for your automatically. The Ad size meta tag is defined within the <head> tag. For example:
-```
-<meta name="ad.size" content="width=300,height=250">
-```
-* Macro replacement is only performed on the index.html. Placing macros into referenced javascript files is not supported.
+* If you include an `ad.size` meta tag in the ad when you upload it, the system will automatically detect and choose the correct ad size for you automatically. The ad size meta tag is defined within the `<head>` section of the `index.html`. For example:
+
+  ```
+  <meta name="ad.size" content="width=300,height=250">
+  ```
+* Macro replacement is only performed on the `index.html`. Placing macros into referenced javascript files is not supported.
 * Expandable ads are not supported.
-* When creating ads with the Google Web Designer you must select Non-Google Ad from the Environment drop down list when creating the ad. DoubleClick, AdMob and AdWords environment ads are not supported.
-* Mobile Ad Networks (MoPub) must use target=”\_blank” on links due to their limited support for IFrames. See here for further details: https://dev.twitter.com/mopub-demand/marketplace/iframe
```

Comment:
I could not find any current information from AppLovin concerning this. The old tweet does not exist and any old MoPub dev docs are no longer available.